ports/servoshell/Cargo.toml

```diff
@@ -106,12 +106,12 @@ serde_json = { workspace = true }
 surfman = { workspace = true, features = ["sm-angle-default"] }
 
 [target.'cfg(not(any(target_os = "android", target_env = "ohos")))'.dependencies]
-accesskit_winit = "0.23"
+accesskit_winit = "0.27"
```

Comment:
Is not using accesskit_winit 0.28 a deliberate choice here?

Reply:
Yes, I tried it before but, accesskit_winit@0.28 uses accesskit@0.20, but egui expects accesskit@0.19 and we get compilation errors.